### PR TITLE
demo-import fix

### PR DESCRIPTION
**Pull request by Xiaosha61**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -6,15 +6,19 @@
...
     "test/advices/async_advices.spec.ts",
-    "test/core/pointcut.spec.ts"
+    "test/core/pointcut.spec.ts",
+    "demo/index.ts"
   ],
   "angularCompilerOptions": {
     "skipTemplateCodegen": true
   }
-}
+}
```
Comment (mgechev) [inline tsconfig.json]: Would you add a newline

File: demo/index.ts [hunk 2]
```diff
@@ -1,13 +1,13 @@
-import {Wove, Metadata, MethodMetadata, beforeMethod, afterMethod, beforeGetter, afterSetter, afterGetter, beforeStaticMethod} from '../lib/aspect';
+import { Wove, Metadata, MethodMetadata, beforeMethod, afterMethod, beforeGetter, afterSetter, afterGetter, beforeStaticMethod } from '../lib/index';
```
Comment (SamVerschueren) [inline demo/index.ts]: `from '../lib';`

File: tsconfig.json (hunk 1 above)
Comment (Xiaosha61) [inline tsconfig.json]: Some auto-formatted stuffs...

**Comment by mgechev** [comment]
@Xiaosha61 thank you for the PR!

**Pull request merged**